pbj-core/buildSrc/src/main/kotlin/com.hedera.pbj.gradle-plugin.gradle.kts

```diff
@@ -26,9 +26,9 @@ tasks.generateGrammarSource {
 // Do not generate Java Doc for generated antlr grammar
 tasks.withType<Javadoc> { excludes.add("com/hedera/pbj/compiler/impl/grammar/**") }
 
-tasks.register("release-maven-central") {
+tasks.register("releaseMavenCentral") {
     group = "release"
     dependsOn(tasks.named("publishPlugins"))
 }
 
-tasks.register("release-maven-central-snapshot") { group = "release" }
+tasks.register("releaseMavenCentralSnapshot") { group = "release" }
```

Comment:
We can remove this task completely. Gradle will skip this project if there is not task matching when we call `gradle releaseMavenCentralSnapshot`